Mode maintenance (API Json uniquement)

feuloren opened the pull request:
Je vois pas comment faire de test la dessus par contre.

https://github.com/payutc/server/issues/308

apuyou commented on src/Payutc/WebApp.php:
```diff
@@ -22,6 +22,9 @@ public static function createApplication($config)
                 $dispatcher->handleError($e);
             });
             
+            if (Config::get('maintenance'))
```
Les accolades sont obligatoires :)

apuyou commented on src/Payutc/WebApp.php:
```diff
@@ -22,6 +22,9 @@ public static function createApplication($config)
             });
             
+            if (Config::get('maintenance'))
+                throw new \Payutc\Exception\MaintenanceException("Le serveur est en cours de maintenance.");
```
Je dirai plutôt _payutc_ est en cours maintenance, vu que c'est ce qui sera affiché dans les clients, il faut retirer l'ambiguïté de si c'est le serveur du client ou bien payutc tout court.

feuloren committed:
Mode maintenance (API Json uniquement)

feuloren commented:
C'est corrigé.
Exceptionnellement j'ai mis une majuscule à payutc pour que la phrase soit grammaticalement correcte.

feuloren committed:
Payutc > français

c'te lutte pour un bête p minuscule,
je remercie le mode zen de github qui est le seul à fonctionner sur un smartphone.

trecouvr commented:
Je ne vois pas de moyen actuellement pour tester ça
